test/features/test_scurv_estimation.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Software License Agreement (BSD License)
```

Comment:
Please use SPDX Identifiers instead of complete license. See filters/functor_filter for an example.

Also, the copyright is wrong (This file is original work, being added to PCL, right?)

Reply:
License changed. Yes, this file is original work published in a paper, can you suggest the copyright "style" or give some example?

Reply:
Current:
```
Copyright (c) 2018, University of Innsbruck (Antonio J Rodríguez-Sánchez, Tomas Turecek, Alex Melniciuc)
```

Alternatives:
1. Handover the rights of **this implementation** to PCL's foundation
```
Copyright (c) 2018-2020, University of Innsbruck (Antonio J Rodríguez-Sánchez, Tomas Turecek, Alex Melniciuc)
Copyright (c) 2020-, Open Perception, Inc.
```
2. Add yourself as an author, maybe add a reference to the paper and start the copyright with PCL
```
Copyright (x) 2020-, OpenPerception (PCL)

\authors Antonio J Rodríguez-Sánchez, Tomas Turecek, Alex Melniciuc (University of Innsbruck)
```
Or a mixture of the above. Reason for giving the copyright to PCL:
* establish a chain of copyright transfer
* make it easier for PCL to maintain and enforce the IP without being required to contact the original authors